iOS/_Stream Temp/MessageController+Extensions.swift

```diff
@@ -69,6 +69,18 @@ extension MessageController {
         })
     }
 
+    func removeReaction(with type: MessageReactionType) async throws {
+        return try await withCheckedThrowingContinuation({ continuation in
+            self.deleteReaction(type) { error in
```

Comment:
The only thing weird here is that it deletes the type, and not the parent reaction itself. Does the parent reaction linger? no idea.